RecoTracker/FinalTrackSelectors/plugins/TrackListMerger.cc

```diff
@@ -487,7 +487,7 @@ void TrackListMerger::produce(edm::Event& e, const edm::EventSetup& es) {
   }
 
   //DL here
-  if LIKELY (ngood > 1 && collsSize > 1)
+  if LIKELY (ngood > 0 && collsSize > 0)
```

Comment:
This condition controls the cleaning of duplicates across different track collection but also within the same track collection. The former makes sense when at least 2 collection are there but the latter makes sense even when a single collection is used. Even though tracking up till now had multiple iterations, and, hence, multiple track collections, there can exist LST configurations with a single iteration, which would still benefit from the duplicate cleaning within the same track collection. This change allows that to happen.

Reply:
Wouldn't this

> there can exist LST configurations with a single iteration, which would still benefit from the duplicate cleaning within the same track collection

be better addressed within the LST reconstruction itself ?

Reply:
What impact does this change have on tracking performance (timing / throughput) ?